docs/project/2024/teodor_matei.francu/index.md

```diff
@@ -0,0 +1,105 @@
+# Tic-Tac_Toe
+This project aims to emulate the popular game Tic-Tac_Toe on a Rapspberry Pi.
+
+
+:::info
+
+**Author**: Teodor-Matei FRÂNCU \
+**GitHub Project Link**: [link_to_github](https://github.com/UPB-FILS-MA/project-Angery-Stoman)
+
+
+
+## Description
+
+
+In this project I will try to recreate simplified version of a game console of the 1990's. The controller for playing the game will be composed of 5 buttons, 4 of which for navigating the game itself (up down left right) and the last for inputting your move. The game will be displayed using a small LCD display that will run using Embedded-graphics, thereby giving the user a simple yet effective interface for playing the game. To emulate the feel of retro games, a buzzer (or more, if need be) will be used to play a simple 8-bit styled theme and specific sounds for inputting moves and losing or winning the game.
+The game will be played against a bot that is coded using some simple checks and ifs.
+
+
+## Motivation
+
+I've always wanted to delve into game making, and while this may not be much, it's a small step into the world of game making. Tic Tac Toe feels simple enough for someone of my hardware knowledge level to recreate it. Choosing a game as my project also adds a level of difficulty to the journey of creating it, as I have only coded games using html, c++/c# and javascript, a.k.a. only software. This will be the first time any game that I have made has a hardware component to it.
+And as an added bonus I'm quite passionate about video games and I spend most of my free time playing them, so this project is right up my alley.
+
+## Architecture 
+
+![architecture](architecture.png)
+
+### Main Components
+
+Here is the list of all of the main components and a short description of their utility:
+
+-Controller
+This controller is composed of 5 simple buttons, 4 of them are for inputting directions (up down left and right) and one is for imputting your move (placing down x or 0 on the playing field)
+
+-Microcontroller
+The brains of this project, responsible with interpreting the input of the controller and sending the appropriate data to be displayed by the LCD display and to the buzzers.
+
+-Display
+Comprised of an lcd display that will present the user the game itself, a 3x3 matrice that both the player and the computer will play on.
+
+-Buzzers
+Used to play different sounds according to the users inputs and the state of the game (loss/win/idle)
+
+### Connections
+
+-Controller to Microcontroller:
```

Comment:
I don't think this renders correctly, please verify this by running `npm start`.